packages/mobile/src/screens/feed-screen/FeedScreen.tsx

```diff
@@ -57,7 +56,6 @@ export const FeedScreen = () => {
           ListFooterComponent={
             <EndOfLineupNotice description={messages.endOfFeed} />
           }
-          LineupEmptyComponent={<EmptyFeedSuggestedFollows />}
```

Comment:
Love getting rid of the complexity!
Do we need to replace this with anything?

Reply:
eventually probably, but lets address that when we are fully react query, should be easy to lift the artist selection from sign-up